WireUI/Tests/WireMainNavigationUITests/Containers/MainTabBarControllerTests.swift

```diff
@@ -23,67 +23,72 @@ import XCTest
 
 final class MainTabBarControllerTests: XCTestCase {
 
+    private var sut: MainTabBarController<
+        MockConversationListViewController,
+        MockConversationViewController<PreviewConversationModel>
+    >!
     private var snapshotHelper: SnapshotHelper!
 
-    override func setUp() {
+    @MainActor
+    override func setUp() async throws {
+        sut = .init()
         snapshotHelper = .init()
             .withSnapshotDirectory(SnapshotTestReferenceImageDirectory)
     }
 
     override func tearDown() {
         snapshotHelper = nil
+        sut = nil
     }
 
     @MainActor
-    func testConversationListAndConversationIsInstalled() throws {
+    func testConversationListIsInstalled() throws {
         // Given
         let conversationList = MockConversationListViewController()
-        let conversation = UIViewController()
-        let sut = MainTabBarController<MockConversationListViewController>()
 
         // When
-        sut.conversations = (conversationList, conversation)
+        sut.conversationList = conversationList
 
         // Then
-        let navigationController = try XCTUnwrap(sut.viewControllers?[1] as? UINavigationController)
-        XCTAssertEqual(navigationController.viewControllers, [conversationList, conversation])
+        let navigationController = try XCTUnwrap(sut.viewControllers?[1] as? UINavigationController) // TODO: [WPB-6647] fix index for navigation overhaul
```

Comment:
still valid TODO?

Reply:
Yes, since this PR merges only parts of the navigation overhaul to develop.
The remaining changes are part of the epic pr https://github.com/wireapp/wire-ios/pull/1361 which should remove all the TODOs with the ticket number `WPB-6647`.